[2025-02-12T16:03:18.140+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2025-02-09T17:00:00+00:00 [queued]>
[2025-02-12T16:03:18.148+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2025-02-09T17:00:00+00:00 [queued]>
[2025-02-12T16:03:18.148+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 4
[2025-02-12T16:03:18.163+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): generate_customer_dim_data> on 2025-02-09 17:00:00+00:00
[2025-02-12T16:03:18.170+0000] {standard_task_runner.py:57} INFO - Started process 3039 to run task
[2025-02-12T16:03:18.173+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'customer_dim_generator', 'generate_customer_dim_data', 'scheduled__2025-02-09T17:00:00+00:00', '--job-id', '74', '--raw', '--subdir', 'DAGS_FOLDER/customer_dim_generator_dag.py', '--cfg-path', '/tmp/tmpj5c_1b2w']
[2025-02-12T16:03:18.174+0000] {standard_task_runner.py:85} INFO - Job 74: Subtask generate_customer_dim_data
[2025-02-12T16:03:18.215+0000] {task_command.py:415} INFO - Running <TaskInstance: customer_dim_generator.generate_customer_dim_data scheduled__2025-02-09T17:00:00+00:00 [running]> on host 5879df32f101
[2025-02-12T16:03:18.287+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='hoyseiha@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='customer_dim_generator' AIRFLOW_CTX_TASK_ID='generate_customer_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-09T17:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-09T17:00:00+00:00'
[2025-02-12T16:03:18.294+0000] {logging_mixin.py:151} INFO - CSV file './customer_dim_large_data.csv' with 100 rows has been generated successfully.
[2025-02-12T16:03:18.295+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-12T16:03:18.304+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=customer_dim_generator, task_id=generate_customer_dim_data, execution_date=20250209T170000, start_date=20250212T160318, end_date=20250212T160318
[2025-02-12T16:03:18.345+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-12T16:03:18.473+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
